opentreemap/treemap/js/src/lib/inlineEditForm.js

```diff
@@ -19,6 +19,7 @@ exports.init = function(options) {
         form = options.form || section + ' form',
         $section = $(section),
         edit = options.edit || '.editBtn',
+        $edit = $(edit),
```

Comment:
Okay, then Mike's concern is unfounded.

@jwalgran, the review is in your hands.